.github/workflows/langchain_experimental_ci.yml

```diff
@@ -13,16 +13,56 @@ on:
       - 'libs/experimental/**'
   workflow_dispatch:  # Allows to trigger the workflow manually in GitHub UI
 
+env:
+  POETRY_VERSION: "1.5.1"
+  WORKDIR: "libs/experimental"
+
 jobs:
   lint:
     uses:
       ./.github/workflows/_lint.yml
     with:
       working-directory: libs/experimental
     secrets: inherit
+
   test:
     uses:
       ./.github/workflows/_test.yml
     with:
       working-directory: libs/experimental
     secrets: inherit
+
+  # It's possible that langchain-experimental works fine with the latest *published* langchain,
```

Comment:
So this means we run tests twice now?

Reply:
Yes, we test once with the langchain in the sibling directory, and once with the langchain version in the lockfile. I'm going to set up a bit of automation soon to make sure the lockfile's langchain version is the most recent one published on PyPI.

This lets us be sure that `langchain-experimental` works both with the currently-released `langchain` on PyPI as well as with any changes in the upcoming release of `langchain`.

The two sets of tests run in parallel and only take about a minute each, so this won't noticeably change CI time.